docs/dev/developer-guides/README.md

```diff
@@ -4,14 +4,14 @@ Take a deeper dive into the zkSync ecosystem.
 
 ## Table of contents
 
-- [Quickstart](./developer-guides/hello-world.md)
-- [System contract](./developer-guides/contracts/system-contracts.md)
-- [Contract deployment](./developer-guides/contracts/contracts.md) 
-- [Contract verification](./developer-guides/contracts/contract-verification.md)
-- [Transaction finality](./developer-guides/transactions/transactions.md)
-- [Account abstraction](./developer-guides/transactions/aa.md)
-- [Fee mechanism](./developer-guides/fee-model.md)
-- [Bridging of funds](./developer-guides/bridging/bridging-asset.md)
-- [L1 / L2 Interoperability](./developer-guides/bridging/l1-l2-interop.md)
-- [L1 / L2 communication](./developer-guides/bridging/l1-l2.md)
-- [L2 / L1 communication](./developer-guides/bridging/l2-l1.md)
+- [Quickstart](./hello-world.md)
+- [System contract](./contracts/system-contracts.md)
+- [Contract deployment](./contracts/contracts.md) 
+- [Contract verification](./contracts/contract-verification.md)
+- [Transaction finality](./transactions/transactions.md)
+- [Account abstraction](./transactions/aa.md)
+- [Fee mechanism](./fee-model.md)
+- [bridging of funds](./bridging/bridging-asset.md)
```

Comment:
Should be "Bridging" with capitals in the link

Reply:
looks like it got reverted somehow: https://github.com/matter-labs/zksync-web-v2-docs/pull/108/commits/c5818f563724b48e97746ed468eca93900faf98d#diff-8af04b40623f9f2c3be342ff05037fa034ca61f2cc921a61a9322823e4082d19R26
will fix it